main.go

```diff
@@ -102,7 +102,6 @@ func main() {
 		Client:  mgr.GetClient(),
 		Scheme:  mgr.GetScheme(),
 		Kclient: kclient,
-		Log:     ctrl.Log.WithName("controllers").WithName("KeystoneAPI"),
```

Comment:
FYI for the reviewers: This information is automatically part of the log line as `"controller": "keystoneapi"` so we are not loosing information by removing it. However if we want to have it in the name of the logger not as a parameter then we can adapt Reconile() to add this information to the name of logger in the context.

Reply:
I personally find it harder to parse the log quickly if the source information is not at the beginning and alignment for that information changes  depending on the message size, but maybe I just have to get used to, but I think I'd prefer to have it included. Lets see what others think about this

Reply:
How does that look ?  : 

It starts with "Controllers.KeystoneAPI" from the GetLog WithNames and adds additional fields from the controller context

```bash
2023-05-18T02:00:27+03:00       INFO    Controllers.KeystoneAPI Reconciling Service     {"controller": "keystoneapi", "controllerGroup": "keystone.openstack.org", "controllerKind": "KeystoneAPI", "KeystoneAPI": {"name":"keystone","namespace":"openstack"}, "namespace": "openstack", "name": "keystone", "reconcileID": "512a4d9b-f31d-4fa4-a4cc-cd6c13e4455d"}
2023-05-18T02:00:28+03:00       INFO    Controllers.KeystoneAPI Reconciled Service init successfully    {"controller": "keystoneapi", "controllerGroup": "keystone.openstack.org", "controllerKind":"KeystoneAPI", "KeystoneAPI": {"name":"keystone","namespace":"openstack"}, "namespace": "openstack", "name": "keystone", "reconcileID": "512a4d9b-f31d-4fa4-a4cc-cd6c13e4455d"}
```

Reply:
It looks good to me.

Reply:
+1 from me as well, thanks!